### PR TITLE
Flock sentinel changes

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[BALANCE]
## About the PR 
Refactors sentinels slightly, buffs their charge rate from 5 -> 20 per process tick, nerfing their range and damage a bit to compensate.
Also makes two sentinels appear with the initial flock rift spawn.

## Why's this needed? 
Sentinels fired _incredibly_ slowly before, this should hopefully make them a little more viable without being perma-stunny and the two free sentinels should help flock's early game a little.

### CODE REVIEW

**Review by amylizzle — APPROVED** [review]

**Comment by TobleroneSwordfish** [comment]
after testing this some more I'm drafting it because it scares me

**Comment by FlameArrow57** [comment]
Would think maybe stuff related to adding on rift spawn should be put off until design doc, but idk

**Comment by Sovexe** [comment]
!merge_upstream

File: code/obj/flock/structure/sentinel.dm [hunk 1]
```diff
@@ -62,22 +68,22 @@
 				charge_status = CHARGING//if its losing charge and suddenly theres energy available begin charging
 			if(CHARGING)
 				if(icon_state != "sentinelon") icon_state = "sentinelon"//forgive me
-				src.charge(5)
+				src.charge(charge_per_tick)
```
Comment (Sovexe) [inline code/obj/flock/structure/sentinel.dm:71]:
```suggestion
				src.charge(charge_per_tick * mult)
```

**Pull request merged**